src/kernel/task.zig

```diff
@@ -23,14 +22,14 @@ extern var KERNEL_STACK_START: *u32;
 pub const EntryPoint = usize;
 
 /// The bitmap type for the PIDs
-const PidBitmap = if (is_test) ComptimeBitmap(u128) else ComptimeBitmap(u1024);
+const PidBitmap = if (is_test) bitmap.Bitmap(true, usize, 128) else bitmap.Bitmap(true, usize, 1024);
```

Comment:
Was there a purpose for having difference bitmaps for testing and for runtime as it does seem a bit odd.

Reply:
I don't remember actually. The tests pass after removing the special case :)